### PR TITLE
Improve SAML setup and troubleshooting

### PR DESCRIPTION
**Pull request by jvlcek**
Fixes https://github.com/ManageIQ/manageiq.org/issues/763

This PR:
Updates the `auth-overview` blog post, to include screen shots for the required SAML Identity Provider, `client` mappers.
Updates the `troubleshooting-auth` blog post to include some tips and pointers for troubleshooting SAML related failures. These updates address an issue recently encountered by customers that had resulted in a support escalation. The hope is that this update will help alleviate similar escalations in the future.

### CODE REVIEW

**Comment by jvlcek** [comment]
@abellotti and @cybette Please review

File: site/_posts/2018-01-31-troubleshooting-auth.md [hunk 1]
```diff
@@ -110,18 +110,85 @@ ManageIQ authentication related configuration data is maintained in the ManageIQ
...
+
+  - Hostname must be DNS resolvable.
+
+    SAML requires DNS resolvable hostnames, IP addresses and unresolvable hostnames will not work.
+
+    Ensure the hostname assigned to the ManageIQ appliance is resolvable on the SAML identity provider and the
+    client running the browser used to access the ManageIQ appliance UI.
+
+    Also ensure the hostname assigned to the SAML IdP is resolvable on the ManageIQ appliance and the
+    client running the browser used to access the MiQ appliance UI.
```
Comment (abellotti) [inline site/_posts/2018-01-31-troubleshooting-auth.md]: MiQ -> ManageIQ

File: site/_posts/2018-01-31-troubleshooting-auth.md [hunk 2]
```diff
@@ -110,18 +110,85 @@ ManageIQ authentication related configuration data is maintained in the ManageIQ
...
+
+
+    If the Apache LogLevel is set to debug, as described above, errors similar to the bellow will be found in the ***/var/www/miq/vmdb/log/apache/ssl_error.log*** log file:
+      ```
+      SubjectConfirmationData was in the past
+        and
+      SSL Library Error: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
+      ```
+
+    The ***DATE(1)*** command can be used on the SAML identity provider and the ManageIQ appliance to confirm the dates on each are syncronized.
```
Comment (abellotti) [inline site/_posts/2018-01-31-troubleshooting-auth.md]: synchronized

File: site/_posts/2018-01-31-troubleshooting-auth.md [hunk 3]
```diff
@@ -110,18 +110,85 @@ ManageIQ authentication related configuration data is maintained in the ManageIQ
...
+
+  - Inorrectly configured SAML ***client*** mappers
```
Comment (abellotti) [inline site/_posts/2018-01-31-troubleshooting-auth.md]: Incorrectly

File: site/_posts/2018-02-01-auth-overview.md [hunk 4]
```diff
@@ -121,15 +121,21 @@ A summary of how to configure ***IPA AD Trust*** with IPA is as follows:
...
 
   The identity provider tool, [Keycloak](http://www.keycloak.org/), has been used to test the SAML functionality.
 
   A summary of how to configure a ManageIQ appliance to work with SAML is as follows:
 
   1. Configuration Apache 
   1. Configure SAML with mellon_create_metadata script
-  1. Configure a Client on the Keycloak Administration Console and configure the required SAML Assertions
+  1. Configure a Client on the Keycloak Administration Console following the [ManageIQ Authentication SAML documentation](http://manageiq.org/docs/reference/latest/auth/saml)
+  1. Add the necessary mappers, paying close attention to the case or each field. Follow these screenshots:
```
Comment (abellotti) [inline site/_posts/2018-02-01-auth-overview.md]: case for each

File: site/_posts/2018-01-31-troubleshooting-auth.md (hunk 3 above)
Comment (jvlcek) [reply site/_posts/2018-01-31-troubleshooting-auth.md]: @abellotti Yes I spell that one ...ino... inc... wrong 
Thank you.

File: site/_posts/2018-02-01-auth-overview.md (hunk 4 above)
Comment (jvlcek) [reply site/_posts/2018-02-01-auth-overview.md]: @abellotti Good catch! Thank you.

**Comment by jvlcek** [comment]
Thank you @abellotti ! I believe I address all the issues you pointed out.

**Comment by jvlcek** [comment]
@cybette Please review and merge if appropriate.

**Pull request merged**